apps/krew-website/src/pages/index.astro

```diff
@@ -4,7 +4,7 @@ import Header from '../components/Header.astro';
 import Footer from '../components/Footer.astro';
 
 ---
-<html lang="en">
+<html class="scroll-smooth" lang="en">
```

Comment:
Nothing is done with this class. Or is it already defined somewhere in the CSS?

Reply:
Tailwind class, for smooth scrolling to anchor